ls1intum/Artemis
# Fix Submissions-Button beeing visible for tutors.

### Checklist

### Motivation and Context
This fixed two places where tutors could see the submissions overview even though this page is only meant for instructors.

### Steps for Testing
1. Go to the Assessment-Dashboard and the programming exercises overview
2. Check that tutors cannot see the submissions button.

### Screenshots
old:
![grafik](https://user-images.githubusercontent.com/26540346/114030750-fcd23a00-987a-11eb-9381-540334ef49c0.png)
![grafik](https://user-images.githubusercontent.com/26540346/114030727-f80d8600-987a-11eb-8632-bddf18e052e8.png)

new:
![grafik](https://user-images.githubusercontent.com/26540346/114030708-f3e16880-987a-11eb-9151-5f8b929107a8.png)
![grafik](https://user-images.githubusercontent.com/26540346/114030682-ee841e00-987a-11eb-9b0b-5cffa6063fef.png)
